lib/ctl.js

```diff
@@ -7,7 +7,7 @@ module.exports = onCtlRequest;
 function onCtlRequest(options, req, callback) {
   debug('request %j', req);
 
-  var instanceId = req.serviceId;
+  var instanceId = req.serviceId || '1';
```

Comment:
why is serviceId being assigned to instanceId? Both exist on the request: https://github.com/strongloop/strong-pm/blob/master/lib/service-manager.js#L113-L114

Maybe `instanceId` needs renaming to `serviceId`? Or maybe in the commands below, it should sometimes be one, and sometimes the other? Everything used to be serviceId, I suspect that it was all changed to be instance, but that change may have been too aggressive.